keras/callbacks.py

```diff
@@ -671,19 +673,25 @@ def set_model(self, model):
         else:
             self.writer = tf.summary.FileWriter(self.log_dir)
 
-        self.saver = tf.train.Saver()
-        self.embeddings = {layer.name: (layer.weights[0],
-                                        os.path.join(self.log_dir, layer.name +
-                                                     '.ckpt'))
-                           for layer in self.model.layers
-                           if layer.name in self.embeddings_layer_names}
-        self.embeddings_metadata_setup()
+        if self.embeddings_freq:
+            self.saver = tf.train.Saver()
+
+            embeddings_layer_names = self.embeddings_layer_names \
```

Comment:
I can re-write an expression not to make it in one line. Something like this:

```python
embeddings_layer_names = self.embeddings_layer_names

if not embeddings_layer_names:
    embeddings_layer_names = [layer.name for layer in self.model.layers
                              if type(layer).__name__ == 'Embedding']
```

But in such an expression I used there's no parenthesis, so new line character is interpreted as the end of expression.

Also I tried to fit in 80 characters width.